uniffi_bindgen/src/bindings/gecko_js/templates/InterfaceTemplate.cpp

```diff
@@ -39,6 +46,14 @@ JSObject* {{ obj.name()|class_name_cpp(context) }}::WrapObject(
   return dom::{{ obj.name()|class_name_cpp(context) }}_Binding::Wrap(aCx, this, aGivenProto);
 }
 
+RefPtr<nsISerialEventTarget> {{ obj.name()|class_name_cpp(context) }}::GetBackgroundTarget() {
+  if (!mBackgroundET) {
+    MOZ_ALWAYS_SUCCEEDS(NS_CreateBackgroundTaskQueue(
+      "SystemInfoThread", getter_AddRefs(mBackgroundET)));
```

Comment:
nit: the name of this queue is copy-pasta and should be updated

Reply:
good catch; thanks